src/bgw/job.h

```diff
@@ -60,7 +68,7 @@ extern TSDLLEXPORT void ts_bgw_job_permission_check(BgwJob *job);
 
 extern TSDLLEXPORT void ts_bgw_job_validate_job_owner(Oid owner, JobType type);
 
-extern bool ts_bgw_job_execute(BgwJob *job);
+extern JobResult ts_bgw_job_execute(BgwJob *job);
```

Comment:
also undo change here. The return types are only true and false. JobResult has 3 states : they are not valid for the function that executes it. A function either executes successfully or fails. It is part of scheduler state.

Reply:
Removed this change.